kvmagent/kvmagent/plugins/block_storage_plugin.py

```
@@ -120,13 +113,25 @@ def translate_absolute_path_from_wwn(wwn):
-
-    return True
+    hb_content = rbd_rw_handler.read(heartbeat_path, fs_id, host_id, 1024)
+    if hb_content is None:
+        return False
+    hb_content = jsonobject.loads(hb_content)
+    hb_content_timestamp = hb_content["heartbeat_time"]
+    if not hb_content_timestamp:
+        return False
+    if hb_content_timestamp == hb_timestamp:
+        return True
+    return False
```

Ruff (0.8.2)

Return the condition `hb_content_timestamp == hb_timestamp` directly

Replace with `return hb_content_timestamp == hb_timestamp`

(SIM103)